src/appengine/handlers/cron/fuzz_strategy_selection.py

```diff
@@ -102,7 +102,25 @@ def _query_multi_armed_bandit_probs(client):
   return client.query(query=BANDIT_PROBABILITY_QUERY).rows
 
 
-def _query_and_upload_strategy_weights(client):
```

Comment:
Were these weights->probs changes intentional? It seems better to leave them as-is to me if not.

Reply:
They were intentional, I wanted to change every mention of 'weights' in the file to 'probabilities' for consistency sake because before it was a mix of the two